### PR TITLE
Rework sidebar with ericz's design

### PR DESCRIPTION
**Pull request by capoferro**

### CODE REVIEW

File: src/actions/repositories.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import {
+  getSubscribedRepositories,
+  getOwnedRepositories,
+  getVisibleRepositories,
+} from "../models/repository"
+
+import { receiveError } from "../actions/ui"
+
+export const FETCH_REPOSITORIES = "FETCH_REPOSITORIES"
```
Comment (erikh) [inline src/actions/repositories.js]: what is this pattern for? Not objecting to it; just not seeing what it's used for.

File: src/components/ManualSubmitForm.js [hunk 2]
```diff
@@ -31,11 +35,13 @@ class ManualSubmitForm extends BaseComponent {
       .fetch("/uisvc/submit?" + qs, { credentials: "include" })
       .then(result => {
         this.setState({ submitting: false })
-        successToast("Submitted!")
+        this.props.uiActions.processSuccess(
+          "Submitted " + this.submitRepository,
```
Comment (erikh) [inline src/components/ManualSubmitForm.js]: nit: this should display the branch name, too.

File: src/components/messagedisplay/MessageDisplay.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        </div>,
+      )
+      this.props.uiActions.clearSuccess()
+    }
+
+    return <span id="error-display" />
+  }
+}
+const mapStateToProps = state => {
+  console.log(state.ui)
```
Comment (erikh) [inline src/components/messagedisplay/MessageDisplay.js]: leftover debugging?

File: src/components/repolist/RepoList.css [hunk 4]
```diff
@@ -0,0 +1,108 @@
+.sidebar-container {
+    background-color: #171D2C;
+    width: 530px;
+    color: #fff;
+    padding-left: 20px;
+    padding-right: 20px;
+/*  font-family: HakenGrotesk-Bold; Pending add to google fonts: https://github.com/google/fonts/issues/1412 */
```
Comment (erikh) [inline src/components/repolist/RepoList.css]: should this be left in here? @invisiblehermit

File: src/models/repository.js [hunk 5]
```diff
@@ -0,0 +1,121 @@
+import axios from "axios"
```
Comment (erikh) [inline src/models/repository.js]: is the port to the swagger client coming in another patch? if not, let's make this axios/query_string pattern a lot more generic.

File: src/models/run.js [hunk 6]
```diff
@@ -2,6 +2,8 @@ import axios from "axios"
 import querystring from "query-string"
 const paths = { list: "/uisvc/runs", count: "/uisvc/runs/count" }
 
+// TODO: move callbacks to end of function parameters in all following functions
```
Comment (erikh) [inline src/models/run.js]: still valid?

File: src/reducers/repositories.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import initialState from "./initialState"
```
Comment (erikh) [inline src/reducers/repositories.js]: assume i'm completely ignorant as to what's happening here. can you explain?

File: src/actions/repositories.js (hunk 1 above)
Comment (capoferro) [reply src/actions/repositories.js]: Which pattern? The bit with declaring the constant?

If so, this is a redux pattern: https://redux.js.org/basics/actions

Comment (erikh) [reply src/actions/repositories.js]: ah ok

File: src/components/repolist/RepoList.css (hunk 4 above)
Comment (capoferro) [reply src/components/repolist/RepoList.css]: The font @invisiblehermit was asking for isn't yet available in google fonts... perhaps we could find another way to get it in. I left that there as a note for when in the future someone wonders why we aren't using that font, but I can delete it for now.

File: src/models/repository.js (hunk 5 above)
Comment (capoferro) [reply src/models/repository.js]: yeah, I decided that there was enough going on in this PR that I shouldn't refactor yet another thing, but it should be in the near future.

File: src/models/run.js (hunk 6 above)
Comment (capoferro) [reply src/models/run.js]: It is. This is another refactor that I decided not to include in this PR, but perhaps this one is small enough that I should just do it.

File: src/reducers/repositories.js (hunk 7 above)
Comment (capoferro) [reply src/reducers/repositories.js]: Initial state is used in reducers to define the starting place for the state in redux, before any events are fired.

https://redux.js.org/basics/reducers

File: src/components/repolist/RepoList.css (hunk 4 above)
Comment (invisiblehermit) [reply src/components/repolist/RepoList.css]: let me find an alternative one

Comment (invisiblehermit) [reply src/components/repolist/RepoList.css]: @capoferro We can use this one: https://fonts.google.com/specimen/Open+Sans?selection.family=Open+Sans. But the font size might need to decrease 1px

**Pull request merged**